webapp/lib/view.dart

```diff
@@ -526,8 +543,42 @@ class ActivePackagesViewPartial {
           ..classes.add('active-package-chart__description')
           ..text = chartData
       );
-    packageElement.append(packageMainContainer);
-    packageElement.append(packageChart);
+    packageElement
+      ..append(packageMainContainer)
+      ..append(packageChart)
+      ..append(
+        new SpanElement()
+          ..classes.add('active-package-menu')
+          ..onClick.listen((event) {
+            event.stopPropagation();
+            var selectedPackageDropdown = new Element.ul()
+              ..classes.addAll(['add-tag-dropdown', 'active-package-dropdown'])
+              ..append(
+                new Element.li()
+                  ..classes.add('add-tag-dropdown__item')
+                  ..text = 'Rename'
+                  ..onClick.listen((event) {
+                    packageNameElement.contentEditable = 'true';
+                    packageNameElement.focus();
```

Comment:
as discussed - can you look into how to set the cursor to appear at the end of the text? (both here and in the dashboard view)

Reply:
Been able to find a working solution for this